### DISCUSSION TITLE
Next JS 10 Image 100% Width and Chakra

### DISCUSSION THREAD

**Discussion by luccazii** [chakra-ui/chakra-ui#2884] [Q&A]
Hi, I'm trying to use NextJS 10 Image Component with Chakra.

The Image Component doesn't get 100% width, while Chakra's Image works fine.
I would like to now if you have any workaround for it.

```
import Image from "next/image";

<Image width={'100%'} height={'auto'} src={'/imgs/lando-norris-mclaren-racing-f1-mcl34-us-grand-prix.jpg'} />
```

Result:
![image](https://user-images.githubusercontent.com/16391722/103144149-a42fc480-4703-11eb-9ff9-4576c1ec0c35.png)

**Comment by luccazii** [comment]
Everything seems to be fine using only img or Chakra Image Component.

```
import { Image } from "@chakra-ui/react"

<Image width={'100%'} height={'auto'} src={'/imgs/lando-norris-mclaren-racing-f1-mcl34-us-grand-prix.jpg'} />
```

![image](https://user-images.githubusercontent.com/16391722/103144180-0c7ea600-4704-11eb-814d-47528822d3a0.png)

**Reply by MustaphaAlioglou** [reply]
yes but it doesnt optimize the image.

**Comment by chr-ge** [comment]
Take a look at https://github.com/chakra-ui/chakra-ui/discussions/2475 or https://github.com/chakra-ui/chakra-ui/discussions/2558
